README.md

```diff
@@ -4,39 +4,53 @@
 [![Docker Repository on Quay](https://quay.io/repository/redhat-developer/devconsole-operator/status "Docker Repository on Quay")](https://quay.io/repository/redhat-developer/devconsole-operator)
 [![Docker Repository on Quay](https://quay.io/repository/redhat-developer/operator-registry/status "Docker Repository on Quay")](https://quay.io/repository/redhat-developer/operator-registry)
 
-DevConsole operator enables a developer-focused view in OpenShift 4.
-It provides a view switcher to transition between the traditional
-Kubernetes cluster administration console referred to as
-Administrator, to this new Developer perspective.
+## Overview
+The DevConsole operator enables a developer-focused view in the OpenShift 4 web console.
+It provides a view switcher to transition between the traditional Kubernetes cluster administration console referred to as Administrator, to a new Developer perspective.
 
-This new Developer perspective provides a high-level abstraction over
-Kubernetes and OpenShift primitives to allow developers to focus on
-their application development.
+This new Developer perspective provides a high-level of abstraction over Kubernetes and OpenShift primitives to allow developers to focus on their application development.
 
 ## Key Features
 
-The Developer perspective is still under active development.  These
-are the main features that are getting developed:
+The Developer perspective is under active development. These
+are the main features being developed:
```

Comment:
Here GitHub is highlighting the change correctly because the line breaks were intact.